.travis.yml

```diff
@@ -34,7 +34,7 @@ install:
   - pip install python-ldap
   - pip install unidecode
   - printf '[options]\n\nrunning_env = dev' > ${HOME}/.openerp_serverrc
-  - ln -s server_environment_files_sample ./server_environment_files
+  - ln -s /home/travis/build/OCA/server-tools/server_environment_files_sample /home/travis/build/OCA/server-tools/server_environment_files
```

Comment:
Use `${TRAVIS_BUILD_DIR}/` instead of `/home/travis/build/OCA/server-tools/`
This variable support:
 - Posible future change of name of this repository
 - Compatibility with travis builds from forks